examples/kustomization/sts-example/sample-data/mc-job-setup-bucket.yaml

```diff
@@ -35,7 +35,8 @@ spec:
   backoffLimit: 5
   template:
     spec:
-      restartPolicy: OnFailure
+      serviceAccountName: mc-job-sa
+      restartPolicy: Never
```

Comment:
If Job and tenant are created at the same time, it won't execute all the time here, is this expected?

Reply:
that's rght, having `restartPolicy: OnFailure` will retry Job if tenant not ready yet. This was working before.